src/viur/core/bones/base.py

```diff
@@ -319,6 +319,18 @@ def __init__(
 
         self.compute = compute
 
+    @property
+    def descr(self):
+        if self._descr:
+            return self._descr
+
+        if self.skel_cls:
+            for name, bone in self.skel_cls.__boneMap__.items():
+                if bone is self:
+                    return name
```

Comment:
Not tested, but if `self.skel_cls` is set, `self.name` should be set as well. That should held the attribute name. Can you confirm this?

```suggestion
        if self.name:
            return self.name
```

Reply:
yes! I was a little bit lost yesterday...